components/service/glean/src/main/java/mozilla/components/service/glean/error/ErrorRecording.kt

```diff
@@ -123,6 +123,12 @@ object ErrorRecording {
             sendInPings = metricData.sendInPings
         )
 
-        return errorMetric.testGetValue(usePingName)
+        // Do not throw a NPE if no error was reported. Return 0
+        // instead.
+        return if (errorMetric.testHasValue(usePingName)) {
+            errorMetric.testGetValue(usePingName)
+        } else {
+            0
+        }
```

Comment:
Just a note -- let's not forget to port this to glean-core.

Reply:
After our discussion over IRC, I dropped this part. It's no longer needed, since we're now reporting an error for start()/stop()/start()/stop().